dev-utils/example-dev-conf.json

```diff
@@ -33,8 +37,8 @@
                 "value": "pPxJH5wUamXcCvSxgywzmANuO0UNR5x3nCbUzsrCeX8="
             },
             {
-                "keypackage": "RklYTUU="
+                "keypackage": "generate keypackage. for e.g.) `ra-sp-server --quote-type Unlinkable --ias-key $IAS_API_KEY --spid $SPID &`,`dev-utils keypackage generate --path ./mls.sgxs --output key.txt` "
```

Comment:
there's that `test_dev_config_file` unit test that tries to make sure the "example" file is somewhat valid -- so you may need to include some example keypackage, or at least some base64 payload.

Reply:
ok, i'll replace with valid one